zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -546,26 +546,28 @@ pub(crate) mod tests {
         }])
         .unwrap();
 
+        // TODO: This test was originally written to use the pre-zip-313 fee rule
+        // and has not yet been updated.
+        #[allow(deprecated)]
```

Comment:
I'm confused as to why lack of this did not block the merge of the previous code.

Reply:
It was because of the `#[allow(deprecated)]` on `mod tests`.